src/Core/Managed/Net45/Extensibility/Implementation/RichPayloadEventSource.cs

```diff
@@ -78,6 +83,40 @@ public void Process(ITelemetry item)
         }
 
         /// <summary>
+        /// Record an operation start.
+        /// </summary>
+        /// <param name="operation">The opeation which is about to start.</param>
+        public void ProcessOperationStart(OperationTelemetry operation)
+        {
+            if (this.EventSourceInternal == null)
+            {
+                return;
+            }
+
+            if (this.EventSourceInternal.IsEnabled(EventLevel.Informational, Keywords.Operations))
+            {
+                operationStartStopHandler(operation, EventOpcode.Start);
+            }
+        }
+
+        /// <summary>
+        /// Record an operation stop.
+        /// </summary>
+        /// <param name="operation">The operation which has just stopped.</param>
+        public void ProcessOperationStop(OperationTelemetry operation)
```

Comment:
Yes, Stop is designed to be paired with Start. It has different keywords and opcodes.
Yes, order is significant (Start has to be before Stop)
Yes, I do see scenarios where we only listen to Start/Stop - which is why I use a different keyword for them.